mlflow/models/model.py

```diff
@@ -556,14 +556,22 @@ def get_model_info(self):
         )
 
     def get_tags_dict(self):
-        tags = {}
         result = self.to_dict()
 
-        for key, value in result.items():
-            if key == "flavors":
-                tags["flavors"] = {k: v for k, v in value.items() if k != "config"}
```

Comment:
interesting, the old code seems to exclude 'config' key, why doesn't it work ?

Reply:
CC @harshilprajapati96 thoughts? anything I missed ?

Reply:
result is something like 
```
{
  "flavors": {
            "tf": "flavor body",
            "python_function": {"config": {"a": 1}, "code": "code"},
        }
}
```
The code I had earlier was only looking for "config" one level down instead of 2 levels down under python_function or similar